config/task/segmentation.yaml

```diff
@@ -1,2 +0,0 @@
-task_name: segmentation
-path_task_function: ${mode}_segmentation.main
```

Comment:
@CharlesAuthier let me know if there was a particular reason you wanted it like this.